src/main/java/org/osiam/addons/administration/controller/LoginController.java

```diff
@@ -31,6 +30,8 @@
     @Inject
     private GeneralSessionData generalSessionData;
 
+    private boolean showAuthCodeErrorMessage = false;
```

Comment:
And that's okay! I think I just need to set this flag only to true if an error happened and [not to false every time](https://github.com/osiam/addon-administration/pull/127/files#diff-b0d434b7a328445c89f11064727878ebR43)